interfaces/IBF-dashboard/src/app/components/disaster-type/disaster-type.component.ts

```diff
@@ -42,8 +42,7 @@ export class DisasterTypeComponent implements OnInit, OnDestroy {
         ({ activeTrigger }) => activeTrigger,
       );
       if (activeDisasterType) {
-        this.selectedDisasterType =
-          DisasterTypeKey[activeDisasterType.disasterType];
```

Comment:
@SwatiTh the problem here was that for 'heavy-rain' (unlike for all other disasters) the key of DisasterTypeKey is not the same as value (heavyRain instead of heavy-rain), so you couldn't use the code this way.

Reply:
Got our point @jannisvisser, Thanks